packages/polaris-typeorm/src/subscribers/common-model-subscriber.ts

```diff
@@ -114,4 +134,14 @@ export class CommonModelSubscriber implements EntitySubscriberInterface<CommonMo
             CommonModelSubscriber.ncHandler.initNotificationCenterConfigurations(logger, ncConfig);
         }
     }
+
+    private static setUpnOfEntity(entity: CommonModel, context: PolarisGraphQLContext) {
+        if (context?.requestHeaders) {
+            const id = context?.requestHeaders?.upn || context?.requestHeaders?.requestingSystemId;
```

Comment:
check if you should switch:
context?.requestHeaders?.upn ?? context?.requestHeaders?.requestingSystemId;